src/aws_utils/cloudwatch_utils.rs

```diff
@@ -1,11 +1,53 @@
 pub mod cloudwatch_utils {
+    pub enum DimensionName {
+        Executor,
+    }
+    impl AsRef<str> for DimensionName {
+        fn as_ref(&self) -> &str {
+            match self {
+                DimensionName::Executor => "Executor",
```

Comment:
`"Executor"` should these be constants?

Reply:
good call, converted all strs to constants